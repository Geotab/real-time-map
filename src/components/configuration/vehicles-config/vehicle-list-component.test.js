import React from "react";
import {
  VehicleListComponent
} from "./vehicle-list-component";
import {
  render,
  mount,
  shallow
} from "enzyme";

let props;
beforeEach(() => {
  props = {
    vehicleDisplayList: [{
        id: "1",
        name: "test device",
        color: {
          r: 1,
          g: 2,
          b: 3,
          a: 4
        }
    }]
  };
});


test("should successfully render the component and match screenshot", () => {
      const component = shallow( < VehicleListComponent {
          ...props
        }
        />);
        expect(component).toMatchSnapshot();
      });