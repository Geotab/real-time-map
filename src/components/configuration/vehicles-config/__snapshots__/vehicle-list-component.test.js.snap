// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should successfully render the component and match screenshot 1`] = `
<li
  className="mdc-list-item"
  key="1"
>
  <span
    className="RTM-iconSquare mdc-list-item__graphic material-icons filterIcon hideConfig"
    onClick={[Function]}
    title="Hide/Show"
  />
  <span
    className="mdc-list-item__graphic material-icons vehicle"
    onClick={[Function]}
    title="Fly to Car on Map"
  />
  <span
    className="RTM-ConfigListItem mdc-list-item__text"
    id="RTMnode-1"
  >
    test device
  </span>
  <span
    className="mdc-list-item__meta material-icons"
    onClick={[Function]}
  />
</li>
`;
